### DISCUSSION TITLE
Disable Profile(preferences) for ONE user

### DISCUSSION THREAD

**Discussion by flopp999** [grafana/grafana#51777] [General]
I would like to be able to have a user that doesn't have access to preferences(change name, password etc.)
I know I can disable Profile for all user but how to do with only ONE user?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
